### DISCUSSION TITLE
Could not post a message to ask for help

### DISCUSSION THREAD

**Discussion by AJamesPhillips** [community/community#55241] [Discussions] [answered]
### Select Topic Area

Bug

### Body

I had created a brand new user but I could not (and still can not) post a question on the discussion forum as the button is missing:

<img width="600" alt="image" src="https://github.com/community/community/assets/526509/85aad37f-a432-42da-ac8b-0aeeb8d1c35d">

This works fine for my older GitHub accounts, as in the button is present to click on.

**Comment by hardal7** [comment]
Hi Alexander, I know this isn't exactly a fix but does this link work for you? 
https://github.com/orgs/community/discussions/new/

**Reply by AJamesPhillips** [reply, marked as answer]
Yes https://github.com/orgs/community/discussions/new?category=discussions might work.  I had deleted the effected user before I saw this so now I can't test it to know 🤷

Thanks for your suggestion @hardal7.  Much appreciated.
